feelslikemonday/app/src/main/java/com/example/feelslikemonday/model/MoodEvent.java

```diff
@@ -0,0 +1,102 @@
+package com.example.feelslikemonday.model;
+
+import android.location.Location;
+
+//Todo: Do we give user's the ability to change time and date?
```

Comment:
I dont think the user should be able to change date and time, it's simply the date/time of the post.

Reply:
Fair